pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -398,3 +413,76 @@ func (e *reconcileError) reason() string {
 func newReconcileError(reason string, err error) *reconcileError {
 	return &reconcileError{degradationReason: reason, err: err}
 }
+
+// createSigner creates a signer using the private key from the privateKeyPath
+func createSigner() (ssh.Signer, error) {
+	privateKeyBytes, err := ioutil.ReadFile(wkl.PrivateKeyPath)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to find private key from path: %v", wkl.PrivateKeyPath)
+	}
+
+	signer, err := ssh.ParsePrivateKey(privateKeyBytes)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to parse private key: %v", wkl.PrivateKeyPath)
+	}
+	return signer, nil
+}
+
+// createUserDataSecret creates a secret 'windows-user-data' in 'openshift-machine-api'
+// namespace. This secret will be used to inject cloud provider user data for creating
+// windows machines
+func (r *ReconcileWindowsMachineConfig) createUserDataSecret() error {
```

Comment:
Please keep this along with rest of the `ReconcileWindowsMachineConfig` i.e. before `ReconcileError` struct and methods.